ionicons.podspec

```diff
@@ -10,6 +10,7 @@ Pod::Spec.new do |s|
   s.platform     = :ios, '5.0'
   s.tvos.deployment_target = "9.0"
   s.source_files = 'ionicons/**/*.{h,m}'
+  s.exclude_files = 'ionicons/ioniconsTests/**/*.{h,m}'
```

Comment:
Thanks for finding this + contributing this PR! I looked into it a bit and _think_ we might be able to just specify
`s.exclude_files = 'ionicons/ioniconsTests/` to ignore the whole test directory altogether. I'll give this branch a test install before merging but should be able to get to it tomorrow

Reply:
Sounds good.

Reply:
Commit updated.